src/app.py

```diff
@@ -152,7 +152,7 @@ def job_cancel(job_id: str) -> Response:
 
 @app.route("/api/user/settings")
 def user_settings() -> Response:
-    return jsonify({"can_register": True, "plugin_name": "Redis",})
+    return jsonify({"can_register": True, "plugin_name": "Redis"})
```

Comment:
You sure you want to update app.py in this PR?

Reply:
Actually yes - I know that this PR is a mixed bag of everything, but the process went like:

 - I documented tools that we use routinely on our projects
 - I noticed that flake8 test is missing from the .gitlab-ci.yml file
 - I've added it, and it started failing due to a dangling comma

I know that this PR should be split to at least three smaller ones :sweat: .